<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFileDialog</class>
 <widget class="QDialog" name="NewFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>New Item</string>
  </property>
  <property name="windowIcon">
   <iconset resource="aide_resource.qrc">
    <normaloff>:/images/images/AIDE_Icon_small.png</normaloff>:/images/images/AIDE_Icon_small.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="DialogContainer">
    <item>
     <widget class="QLabel" name="DialogHeader">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New File Creator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="FileNameContainer">
      <item>
       <widget class="QLabel" name="FileNameLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>FileName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FileNameInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="LocationContainer">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="LocationLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LocationInput">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DirectoryBrowserButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="DialogButtonHolder">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>400</number>
      </property>
      <property name="rightMargin">
       <number>80</number>
      </property>
      <item>
       <widget class="QPushButton" name="CreateButton">
        <property name="text">
         <string>Create File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="aide_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewFileDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
